src/ripple/app/ledger/Ledger.cpp

```diff
@@ -1000,6 +972,63 @@ Ledger::unchecked_replace(
     auto const ours = std::move(sle);
 }
 
+std::size_t
+Ledger::txCount() const
+{
+    // Always zero for closed ledgers.
+    return 0;
+}
+
+bool
+Ledger::txExists (uint256 const& key) const
+{
+    return txMap().hasItem (key);
+}
+
+void
+Ledger::txInsert (uint256 const& key,
+    std::shared_ptr<Serializer const
+        > const& txn, std::shared_ptr<
```

Comment:
Nit: I find the old indentation easier to read. I don't object to the short line, but putting 2/3 of a declaration per line is harder to read. Put the start of the second parameter on it's own line?